### ISSUE TITLE
Typo in `numerationOptions`

### ISSUE THREAD

**Issue by samreid** [phetsims/unit-rates#204]
Discovered during https://github.com/phetsims/tasks/issues/730#issuecomment-370860967

DoubleNumberLine.js has:

```js
numerationOptions: null, // {*} options specific to the rate's numerator, see below
```

Whereas elsewhere in the file it is referred to as `numeratorOptions`

**Comment by pixelzoom** [comment]
Good catch. Ditto in MarkerEditorNode, MarkerNode, ShoppingScene.

**Comment by pixelzoom** [comment]
The typo appeared only in the default options, in all cases: `numerationOptions: null`.   Since other usages were spelled correct, and the default value was `null`, fixing the name in the options hash had no negative side-effects.   To be certain, I tested the affected components, and compared to the published version (1.0.4).  The numerator looked correct in all cases.

Closing.
